source/help/messaging/sending-messages.rst

```diff
@@ -1,39 +1,48 @@
 .. _sending-messages:
 
-Sending Messages
+Messages
 ================
+ 
+Sending messages and replying to messages are important ways to keep conversation active with your team. You can edit and delete messages after you have sent them. You can also generate and share links to any message in Mattermost.
 
-Messages are composed in the center pane and reply messages are composed in the right-hand sidebar. Depending on your system configuration, you can edit and delete messages after they are sent. You can also generate and share links to any message in Mattermost.
-
-Sending a Message
+Sending Messages
 -----------------
 
-Write a message by typing into the text input box, then press **ENTER** to send it. Use **SHIFT+ENTER** to create a new line without sending a message. To send messages by pressing **CTRL+ENTER**, go to **Main Menu > Account Settings > Send messages on Ctrl + Enter**.
+Press **Enter** to send a message. To use paragraphs, press **Shift + Enter** to create a new line. If you want to, you can change the settings to send messages with **Ctrl + Enter** instead:
+
+-  **Main Menu > Account Settings > Send messages on Ctrl + Enter**
```

Comment:
This sequence is incorrect. When you open Account Settings dialog, there is no **Send messages on Ctrl + Enter**. Additional steps are required.